manifest.json

```diff
@@ -237,6 +235,37 @@
             }
           }
         ]
+      },
+      "includeTwitterMetaTags": {
+        "oneOf": [
+          {
+            "properties": {
+              "includeTwitterMetaTags": {
+                "enum": [false]
+              }
+            }
+          },
+          {
+            "properties": {
+              "includeTwitterMetaTags": {
+                "enum": [true]
+              },
+              "twitterUsername": {
+                "title": "admin/store.twitterUsername.title",
+                "description": "admin/store.twitterUsername.description",
+                "type": "string"
+              },
+              "twitterCard": {
+                "title": "admin/store.twitterCard.title",
+                "description": "admin/store.twitterCard.description",
+                "type": "string",
+                "default": "summary",
+                "enum": ["photo", "summary", "summary_large_image"],
+                "enumNames": ["Photo", "Summary", "Summmary Large Image"]
+              }
+            }
```

Comment:
Where are you including the code of these properties on the code? If you won't need it, remove it.

Reply:
This is are the settings the user can edit in the store settings page as can be seen in the first screenshot for this PR.

Reply:
Sorry, you're right, my mistake